AK/Statistics.h

```diff
@@ -55,6 +55,10 @@ class Statistics {
     T const median()
     {
         quick_sort(m_values);
+        if (size() % 2 == 0) { // If the number of values is even, the median is the arithmetic mean of the two middle values
```

Comment:
Nit-pick: Please move the comment onto its own line, above this one.

Reply:
Okay, done ;)